test/unit_tests/braket/circuits/test_angled_gate.py

```diff
@@ -50,6 +50,12 @@ def test_angle_setter(angled_gate):
     angled_gate.angle = 0.14
 
 
+def test_adjoint_list(angled_gate):
```

Comment:
Can you add a test with an expression included?
```suggestion
def test_adjoint_list_parameterized():
    theta = FreeParameter("theta")
    AngledGate(angle=(theta + 1), qubit_count=1, ascii_symbols=["foo"])
    assert angled_gate.adjoint_list() == [
        AngledGate(angle=-(theta + 1), qubit_count=1, ascii_symbols=["foo"])
    ]
```

Reply:
This would be in addition to the current test. The above is not something to commit